test/test.js

```diff
@@ -0,0 +1,17 @@
+var request = require('request'),
+		assert = require('assert'),
+		base_url = "http://localhost:3000/";
+
+describe("Job portal Index",function(){
+
+	describe("GET /", function(){
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*Origin: SpaceConsistencyBear, Section: `cli`.*

The issue can be fixed by applying the following patch:

```diff
--- a/test/test.js
+++ b/test/test.js
@@ -4,7 +4,7 @@
 
 describe("Job portal Index",function(){
 
-	describe("GET /", function(){
+    describe("GET /", function(){
 
 		it("returns status code 200", function(){
 			request.get(base_url, function(err, response,body){
```